src/Makefile.am

```diff
@@ -613,6 +613,9 @@ endif
 
 if ENABLE_TESTS
 include Makefile.test.include
+else
+check-local:
```

Comment:
Not sure if this going to run first when we run `make check`. I take a look at `check-local` in `src/Makefile.test.include` and compare with test output found out it run after all unit tests have been ran.

Reply:
Makefile.test.include is not included when we configured without tests. This check-local purpose is to fail the tests when we configured without tests.

Reply:
@ultimaweapon Are you ok with Andrey's reply?